src/client/app/app/components/HeaderWithErrorPanel.tsx

```diff
@@ -178,14 +177,14 @@ const HeaderWithErrorPanel: FunctionComponent<OwnProps> = ({ queryStrings, crash
 			<div ref={wrapperRef}>
 				<Header title={intl.formatMessage({ id: 'Header.K9Los' })} changeLocation={goToHomepage}>
 					{visAdminKnapp() && (
-						<Knapp className={styles.knapp} onClick={goTilDriftsmeldingerPanel}>
+						<button className={styles.knapp} onClick={goTilDriftsmeldingerPanel}>
```

Comment:
Berre for å forsikre: Her ønsker du å erstatte Knapp med native html `button`, og ikkje aksel `Button` komponent?

Reply:
jepp, det er i navbaren på toppen så den ekstra stylingen på <Button/> blir bare til bry